tools/wpt/revlist.py

```diff
@@ -0,0 +1,299 @@
+if MYPY:
+    # MYPY is set to True when run under Mypy.
+    from typing import Any
+    from typing import Callable
+    from typing import Dict
+    from typing import List
+    from typing import Text
+    from typing import Union
+
+
+def parser_debug_log_level(**kwargs):
+    # type: (**Any) -> Any
+    debug_level = kwargs.get("log_level", DEFAULT_LOG_LEVEL)
+
+    if isinstance(debug_level, int):
+        return debug_level
+
+    try:
+        return int(debug_level)
+    except Exception:
+        pass
+
+    return debug_level
+
+
+def parser_epoch_step(**kwargs):
+    # type: (**Any) -> int
+    epoch_step = kwargs.get("epoch_step", DEFAULT_EPOCH_STEP)
+    if isinstance(epoch_step, int):
+        return epoch_step
+    if epoch_step.lower() == "three-hourly":
+        return 10800
+    if epoch_step.lower() == "six-hourly":
+        return 21600
+    if epoch_step.lower() == "twelve-hourly":
+        return 43200
+    if epoch_step.lower() == "daily":
+        return 86400
+    if epoch_step.lower() == "weekly":
+        return 604800
+    return int(epoch_step)
+
+
+def parser_epoch_until(**kwargs):
+    # type: (**Any) -> int
+    epoch_until = kwargs.get("epoch_until", DEFAULT_EPOCH_UNTIL)
+    if isinstance(epoch_until, int):
+        return epoch_until
+    if epoch_until.lower() == "now":
+        return DEFAULT_EPOCH_UNTIL
+    return int(epoch_until)
+
+
+def parser_num_revisions(**kwargs):
+    # type: (**Any) -> int
+    num_revisions = kwargs.get("num_revisions", DEFAULT_NUM_REVISIONS)
+    return int(num_revisions)
+
+
+COMMAND_ARGS = {
+    "epoch_until": {
+        "name": "epoch-until",
+        "default": DEFAULT_EPOCH_UNTIL,
+        "help": "Show revisions older than a specific date in UNIX "
+                "timestamp. NOW is the default value. For example: "
+                "'1570345200'.",
+        "parser": parser_epoch_until
+    },
+    "epoch_threshold": {
+        "name": "epoch-threshold",
+        "default": DEFAULT_EPOCH_THESHOLD,
+        "help": "Safety threshold respect now "
+                "(default: %s seconds)." % DEFAULT_EPOCH_THESHOLD,
+        "parser": lambda **kwargs: int(kwargs.get("epoch_threshold",
+                                                  DEFAULT_EPOCH_THESHOLD)),
+        "type": int
+    },
+
+    "epoch_step": {
+        "name": "epoch-step",
+        "default": DEFAULT_EPOCH_STEP,
+        "help": "Regular interval of seconds used to get the tagged revision. "
+                "three-hourly, six-hourly, eight-hourly, twelve-hourly, "
+                "daily, weekly or 3600, 7200, 10800 ...",
+        "parser": parser_epoch_step
+    },
+    "head_revision": {
+        "name": "head-revision",
+        "default": DEFAULT_HEAD_REVISION,
+        "help": "Head revision (default: %s)" % DEFAULT_HEAD_REVISION,
+        "parser": lambda **kwargs: kwargs.get("head_revision",
+                                              DEFAULT_HEAD_REVISION),
+    },
+    "log_level": {
```

Comment:
This is probably helpful when debugging, but a quick scan suggests that other `./wpt` subcommands don't have such an argument, so I suggest omitting it here too.